### DISCUSSION TITLE
The initial setup download keeps failing half way.

### DISCUSSION THREAD

**Discussion by tepputeppu** [w-okada/voice-changer#1429] [Q&A] [answered]
Issue: The initial setup download keeps failing before it finishes.
Version: vcclient_win_cuda_2.0.61-alpha.
Suggestion: Please download the necessary modules one by one instead of all at once(so that it can at least finish downloading one of them), don't start downloading the next module before the previous download is finished. Or provide a manual download link.

**Comment by AnnoyingFox** [comment]
Decompress to a new folder and start over. Make sure that your connection is stable. If it fails to download, you should start over with a new folder.

**Comment by tepputeppu** [comment, marked as answer]
I have figured out how to manually download it. The download links are in the log file
